config/pipeline.yaml

```diff
@@ -0,0 +1,200 @@
+
+  # Clean data, minor refined versions of provided data.
+  source_table: "complaint_database"
+  nlp_features_table: "complaints_nlp"
+  cleaned_features_table: "complaints_clean_categories"
+  nulls_removed_table: "complaints_nulls_removed"
+  clean_table: "complaints_clean"
+  
+  # Split features into training and prediction.
+  train_predict_split: "TrainPredictSplit"
+  features_train_table: "FeaturesTrain"
+  features_predict_table: "FeaturesPredict"
+
+  # Output tables for predictions and failed predictions from the batch
+  # prediction job, ex. numeric column recieved a string.
+  predictions_table: "Predictions"
+  failed_predictions_table: "FailedPredictions"
+  
+  # Evaluation
+  eval_table: "Evaluation"
+  eval_metrics_table: "EvaluationMetrics"
+
+  # AutoML Parameters. It is assumed that no other datasets or models have
+  # been created with these names, or the pipeline will fail. They can be
+  # deleted using the client, or the UI.
+  dataset_display_name: "CFPBDataset"
+  model_display_name: "CFPBModel"
+  automl_compute_region: "us-central1"
+
+  # Final model will be trained on test data, metrics are currently based
+  # on the predict split instead. AutoML Requires at least one row of test data
+
+# Files with templated queries, in file_paths.queries.
+query_files:
+  remove_nulls: "remove_nulls.sql"
+  clean_categories: "clean_categories.sql"
+  combine_tables: "combine_tables.sql"
+  train_predict_split: "train_and_predict_split.sql"
+  prediction_features: "prediction_features.sql"
+  training_features: "training_features.sql"
+
+# Parameters specific to individual SQL pipeline steps.
+query_params:
+
+  removing_nulls:
+    column1: "consumer_complaint_narrative"
+    column2: "company_response_to_consumer"
+    
+  category_cleanup:
+    old_issue1: "Dealing with your lender or servicer"
+    old_issue2: "Dealing with my lender or servicer"
+    new_issue1: "Other"
+    new_issue2: "Dealing with lender or servicer"
+    old_subprod1: "Other (i.e. phone, health club, etc.)"
+    old_subprod2: "Loan"
+    new_subprod1: "Other"
+    new_subprod2: "Other Loan"
+    old_subissue1: "Debt is not yours"
+    old_subissue2: "Debt is not mine"
+    old_subissue3: "Account status"
+    new_subissue1: "Other"
+    new_subissue2: "Incorrect debt"
+    new_subissue3: "Account status incorrect"
+    
+  train_predict_split:
+    train_threshold: 0.8
+    test_threshold: 0.2
+      
+
+# Training parameters for the model.
+model:
+  # See https://cloud.google.com/automl-tables/docs/train for more details
+  # Training objective and maximum train time, early stopping is enabled.
+  train_budget_hours: 1
+  optimization_objective: "MINIMIZE_LOG_LOSS"
+  
+  # See https://cloud.google.com/automl-tables/docs/prepare for more details on
+  # the target and split columns.
+  # Target column to predict
+  target_column: "company_response_to_consumer"
+  # Split dataset into training, validation, and test.
+  split_column: "splitting"
```

Comment:
does this need to match the name of another column in a different query? If so, this should be pulled into that query too, it it should be clear to the user that the config values need to match.

Reply:
It doesn't need to match. Would it better if I change the column name from "splitting" to something else to avoid confusion?